http/src/test/groovy/io/micronaut/http/simple/cookies/SimpleCookie.groovy

```diff
@@ -0,0 +1,61 @@
+/*
+ * Copyright 2021 original authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.micronaut.http.simple.cookies
+
+import io.micronaut.http.cookie.Cookie
+import io.micronaut.http.cookie.SameSite
+import spock.lang.Specification
+
+class SimpleCookieSpec extends Specification {
+
+    def "SimpleCookie.toString() returns a set-cookie compliant string"() {
```

Comment:
This test isn't testing that the string is compliant with the spec. This should instead pass the string into a decoder that would normally fail if it was malformed.

Reply:
It would be possible to use `io.netty.handler.codec.http.cookie.ServerCookieDecoder` to do it, but it is not available in the classpath. Am I allowed to include it as a test dependency?